yjit/src/codegen.rs

```diff
@@ -5683,6 +5710,41 @@ fn gen_send_iseq(
         argc = lead_num;
     }
 
+    if unsafe { get_iseq_flags_has_rest(iseq) } {
+
```

Comment:
```suggestion
    if iseq_has_rest {
```

It's too long to easily figure out whether the variable is in the same function or not, but it seems to be available?